pages/migrations/0106_printorder.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 2.0.2 on 2018-05-21 16:37
+
+from django.db import migrations, models
+import django.db.models.deletion
+import wagtail.core.blocks
+import wagtail.core.fields
+import wagtail.images.blocks
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('wagtaildocs', '0007_merge'),
+        ('wagtailimages', '0019_delete_filter'),
+        ('pages', '0105_privacypolicy'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='PrintOrder',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('intro_heading', models.CharField(max_length=255)),
+                ('intro_description', models.TextField()),
```

Comment:
We determined we will also need the featured provider name, in case the logo is not self-explanatory.